publiq-ui/pub-title.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <component :is="heading"><slot /></component>
+</template>
+
+<script>
+  export default {
+    props: {
+      size: {
+        type: Number,
+        default: 2,
+        validator(val) {
+          return [1, 2, 3, 4].includes(val);
```

Comment:
I'm wondering if it wouldn't be safer to leave out `3` and `4` for now, since we don't have styling for them?
Or if you put them there intentionally, we should probably also add stories for them?

Reply:
No indeed I forgot to remove them from the validator function